### PR TITLE
Fixed bug in Optimetrics parametric analysis (issue #2021)

### LINKED ISSUE #2021: Bug in Optimetrics parametric analysis
### Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### Description of the bug

When defining sweep variations manually (or from .csv files), the native API requires an argument like the following: 
```
oModule.EditSetup("ParametricSetup1", 
	[
		"NAME:ParametricSetup1",
		"IsEnabled:="		, True,
		[
			"NAME:ProdOptiSetupDataV2",
			"SaveFields:="		, True,
			"FastOptimetrics:="	, True,
			"SolveWithCopiedMeshOnly:=", False
		],
		[
			"NAME:StartingPoint"
		],
		"Sim. Setups:="		, ["Setup1"],
		[
			"NAME:Sweeps",
			[
				"NAME:SweepDefinition",
				"Variable:="		, "var1",
				"Data:="			, "nominalvalue_var1",
				"OffsetF1:="		, False,
				"Synchronize:="		, 0
			],
			.......
		],
		[
			"NAME:Sweep Operations",
			"add:="			, [list_of_variables_values_1],
			"add:="			, [list_of_variables_values_2],
			..........
			"add:="			, [list_of_variables_values_n]
		],
		[
			"NAME:Goals"
		]
	])
```

The problems I have found are:

1. Icepak has the key `FastOptimetrics` instead of `CopyMesh` within the `ProdOptiSetupDataV2` dictionary.
2. The multiple `"add"` lines within the `"NAME:Sweep Operations"` section makes it incompatible with the dictionary structure. The generic methods within pyaedt attempt to add multiple `"add"` keys. The same` "add"` key is always overwritten in the dictionary, and only the last list of values remains:
```
{
        .......
        
	"Sweep Operations": {
		"add": [list_of_variables_values_n]
	}
	
       .......
}
```
3. The dictionary related to the parametric setup remains empty when using the `add_from_file` function (in DesignXPloration, the function calls the `ImportSetup `native API in the Optimetrics module). It is therefore impossible to edit the setup at a later time.

### Steps To Reproduce

1. Add a parametric analysis in icepak from file using pyaedt function 
2. Load in pyaedt a file with a table-defined parametric analysis

### Which Operating System are you using?

Windows

### Which Python version are you using?

3.10

### Installed packages

aedttest==0.0.1
aiohttp==3.8.3
aiosignal==1.2.0
anyio==3.6.2
appdirs==1.4.4
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
asgiref==3.5.2
asttokens==2.0.8
async-timeout==4.0.2
attrs==22.1.0
Babel==2.10.3
backcall==0.2.0
beautifulsoup4==4.11.1
bleach==5.0.1
certifi==2022.9.24
cffi==1.15.1
charset-normalizer==2.1.1
clr-loader==0.2.4
colorama==0.4.6
contourpy==1.0.5
cycler==0.11.0
debugpy==1.6.3
decorator==5.1.1
defusedxml==0.7.1
Django==3.2.8
entrypoints==0.4
executing==1.1.1
fastjsonschema==2.16.2
fonttools==4.38.0
frozenlist==1.3.1
idna==3.4
imageio==2.22.2
ipykernel==6.16.2
ipython==8.5.0
ipython-genutils==0.2.0
jedi==0.18.1
Jinja2==3.1.2
joblib==1.2.0
json5==0.9.10
jsonschema==4.16.0
jupyter-server==1.21.0
jupyter_client==7.4.4
jupyter_core==4.11.2
jupyterlab==3.5.0
jupyterlab-pygments==0.2.2
jupyterlab_server==2.16.1
kiwisolver==1.4.4
MarkupSafe==2.1.1
matplotlib==3.6.1
matplotlib-inline==0.1.6
mistune==2.0.4
multidict==6.0.2
nbclassic==0.4.5
nbclient==0.7.0
nbconvert==7.2.2
nbformat==5.7.0
nest-asyncio==1.5.6
networkx==2.8.8
notebook==6.5.1
notebook_shim==0.2.0
numpy==1.23.4
packaging==21.3
pandas==1.5.1
pandocfilters==1.5.0
parso==0.8.3
pickleshare==0.7.5
Pillow==9.2.0
plumbum==1.8.0
prometheus-client==0.15.0
prompt-toolkit==3.0.31
psutil==5.9.3
pure-eval==0.2.2
pyaedt==0.6.25
pycparser==2.21
Pygments==2.13.0
pyparsing==3.0.9
pyrsistent==0.18.1
python-dateutil==2.8.2
pythonnet==3.0.1
pytz==2022.5
pyvista==0.36.1
pywin32==304
pywinpty==2.0.9
pyzmq==24.0.1
requests==2.28.1
rpyc==5.0.1
scikit-learn==1.1.3
scipy==1.9.3
scooby==0.6.0
Send2Trash==1.8.0
six==1.16.0
sklearn==0.0
sniffio==1.3.0
soupsieve==2.3.2.post1
sqlparse==0.4.3
stack-data==0.5.1
terminado==0.17.0
threadpoolctl==3.1.0
tinycss2==1.2.1
tomli==2.0.1
tornado==6.2
traitlets==5.5.0
urllib3==1.26.12
vtk==9.2.2
wcwidth==0.2.5
webencodings==0.5.1
websocket-client==1.4.1
wslink==1.9.0
yarl==1.8.1

### PR DESCRIPTION
**Pull request by wilove997**
Close #2021

### bug n.1

> Icepak has the key `FastOptimetrics` instead of `CopyMesh` within the `ProdOptiSetupDataV2` dictionary.

**Solution**: After import of the base dictionary from OptimetricsTemplates, if app=icepak then the dictionary is corrected

### bug n.2

> The multiple `"add"` lines within the `"NAME:Sweep Operations"` section makes it incompatible with the dictionary structure.

**Solution**: use of a single add key in the dictionary with a list of lists of variations. Changes in LoadAEDTFile and DesignXPloration to account for this ad-hoc treatment

### bug n.3

> The dictionary related to the parametric setup remains empty when using the `add_from_file` function (in DesignXPloration, the function calls the `ImportSetup `native API in the Optimetrics module).
**Solution**: instead of using the native API, emulate it by defining the props dictionary within the function

### CODE REVIEW

**Comment by Samuelopez-ansys** [comment]
@wilove997 @maxcapodi78 I fixed the IPython problem and I also fixed an small bug (it was missing the first point of the table). Please review it and merge it.

File: pyaedt/modules/DesignXPloration.py [hunk 1]
```diff
@@ -1068,9 +1079,60 @@ def add_from_file(self, filename, parametricname=None):
...
+            setup.props["Sweeps"]["SweepDefinition"] = sweep_definition
+
+            args = ["NAME:" + parametricname]
+            _dict2arg(setup.props, args)
+
+            setup.props["Sweep Operations"] = OrderedDict({"add": []})
+            table = []
+            for var_name in csvreader.fieldnames:
+                if var_name != "*":
+                    table.append(firstDataLine[var_name])
```
Comment (MaxJPRey) [inline pyaedt/modules/DesignXPloration.py:1111]:
```suggestion
            first_data_line = next(csvreader)
            setup.props["Sweeps"] = {"SweepDefinition": OrderedDict()}
            sweep_definition = []
            for var_name in csvreader.fieldnames:
                if var_name != "*":
                    sweep_definition.append(
                        OrderedDict(
                            {
                                "Variable": var_name,
                                "Data": first_data_line[var_name],
                                "OffsetF1": False,
                                "Synchronize": 0,
                            }
                        )
                    )
            setup.props["Sweeps"]["SweepDefinition"] = sweep_definition

            args = ["NAME:" + parametricname]
            _dict2arg(setup.props, args)

            setup.props["Sweep Operations"] = OrderedDict({"add": []})
            table = []
            for var_name in csvreader.fieldnames:
                if var_name != "*":
                    table.append(first_data_line[var_name])
```
